data/campaigns/World_Conquest/scenarios/WC_II_scenario.cfg

```diff
@@ -102,12 +102,12 @@ _ "World Conquest 4p" #enddef
         image = {IMAGE_ONE}
         type = mp
         abbrev = _ "WC" + {PLAYERS}\
-        {WC2_CAMPAIGN_DIFFICULTY VERY_EASY {WC2_HUMAN_DIFFICULTY human-peasants/peasant purple} _"Peasant" _"Beginner" 6 2 2 10 yes 0}
-        {WC2_CAMPAIGN_DIFFICULTY EASY {WC2_HUMAN_DIFFICULTY human-loyalists/sergeant black} _"Sergeant" _"Easy" 7 3 2 7 yes 5}
-        {WC2_CAMPAIGN_DIFFICULTY NORMAL {WC2_HUMAN_DIFFICULTY human-loyalists/lieutenant brown} _"Lieutenant" _"Medium" 8 4 2 5 yes 10} {DEFAULT_DIFFICULTY}
-        {WC2_CAMPAIGN_DIFFICULTY HARD {WC2_HUMAN_DIFFICULTY human-loyalists/general orange} _"General" _"Hard" 8 5 2 2 no 13}
-        {WC2_CAMPAIGN_DIFFICULTY VERY_HARD {WC2_HUMAN_DIFFICULTY human-loyalists/marshal white} _"Grand Marshal" _"Challenging" 9 6 2 1 no 17}
-        {WC2_CAMPAIGN_DIFFICULTY NIGHTMARE {WC2_NIGHTMARE_DIFFICULTY} _"Nightmare" _"Expert" 9 7 1 0 no 20}
+        {WC2_CAMPAIGN_DIFFICULTY VERY_EASY {WC2_HUMAN_DIFFICULTY human-peasants/peasant purple} _"Peasant" _"Normal" 6 2 2 10 yes 0}
+        {WC2_CAMPAIGN_DIFFICULTY EASY {WC2_HUMAN_DIFFICULTY human-loyalists/sergeant black} _"Sergeant" _"Challenging" 7 3 2 7 yes 5}
+        {WC2_CAMPAIGN_DIFFICULTY NORMAL {WC2_HUMAN_DIFFICULTY human-loyalists/lieutenant brown} _"Lieutenant" _"Difficult" 8 4 2 5 yes 10} {DEFAULT_DIFFICULTY}
+        {WC2_CAMPAIGN_DIFFICULTY HARD {WC2_HUMAN_DIFFICULTY human-loyalists/general orange} _"General" _"Fanatical" 8 5 2 2 no 13}
+        {WC2_CAMPAIGN_DIFFICULTY VERY_HARD {WC2_HUMAN_DIFFICULTY human-loyalists/marshal white} _"Grand Marshal" _"Torment" 9 6 2 1 no 17}
```

Comment:
"Fanatical" and "Torment" are new strings. While no-one has taken the decision to formally say "no" to this PR, now that the string freeze has started I think it's time to say that no-one's approved it for 1.18, and that the window for merging it into 1.18 has closed.

Reply:
I am personally not happy with "Torment" and "Fanatical" terms being used here. I suggest closing this until we actually find suitable names for the difficulties. Also, 2 more easier and 1 more harder difficulty might be coming soon.